### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by palmerj** [mapbox/tippecanoe#876]
Haven't seen much activity on this repo for about 6 months now. Is it still maintained?

**Comment by jdesboeufs** [comment]
As [it is explained here](https://github.com/mapbox/tippecanoe#zap-mapbox-has-a-new-service-for-creating-vector-tilesets-zap) Mapbox has launched a new hosted and proprietary service. They will probably no longer maintain this repository.
It's time to fork ;)

**Comment by ericfischer** [comment]
Yes, Mapbox is trying to promote the use of MTS instead, so Tippecanoe maintenance is unfortunately considered a distraction at this point.

**Comment by bdon** [comment]
I'm working on new features - focused on alternate input/output formats for now - in a fork at https://github.com/protomaps/tippecanoe ; welcome to chime in on issues on that repo.

**Comment by romankaravia** [comment]
[Something](https://twitter.com/enf/status/1551593091336769543) tells me that there's a new promising fork here: https://github.com/felt/tippecanoe

**Comment by e-n-f** [comment]
Indeed, Felt's support for Tippecanoe development was announced today: https://felt.com/blog/erica-fischer-tippecanoe-at-felt

**Comment by curran** [comment]
Hooray! I'm thrilled to learn of this.
